### PR TITLE
feat: settings re-design

### PR DESCRIPTION
**Pull request by J1za**
## Linear Ticket
https://linear.app/autokitteh/issue/UI-841/settings-re-design
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/locales/en/modals/translation.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
 	},
 	"deleteAccount": {
 		"cancelButton": "Cancel",
-		"deleteButton": "Delete",
+		"deleteButton": "Yes, Delete",
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/locales/en/modals/translation.json]: changed

File: src/components/molecules/copyButton.tsx [hunk 2]
```diff
@@ -12,11 +12,15 @@ import { IconButton } from "@components/atoms";
 import { CopyIcon } from "@assets/image/icons";
 
 export const CopyButton = ({
+	children,
+	classIcon,
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/components/molecules/copyButton.tsx]: reverted

File: src/components/organisms/settings/clientConfiguration.tsx [hunk 3]
```diff
@@ -25,72 +25,78 @@ export const ClientConfiguration = () => {
...
 
 							<CopyButton
-								className="ml-4 rounded-md bg-gray-900 px-3 hover:bg-gray-950"
+								classIcon="fill-gray-750"
+								className="ml-5 gap-2 rounded-full border border-gray-750 px-5 py-3 hover:bg-gray-1050"
 								successMessage={t("getToken.copySuccess")}
 								text={token}
-							/>
+							>
+								<span className="font-averta font-semibold">{t("copy")}</span>
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/components/organisms/settings/clientConfiguration.tsx]: changed

File: src/components/molecules/copyButton.tsx [hunk 4]
```diff
@@ -41,13 +45,18 @@ export const CopyButton = ({
...
+		{ "w-auto h-auto": children },
 		className
 	);
-	const copyButtonIconStyle = cn("h-5 w-5 fill-white", {
-		"size-5": size === "md",
-		"size-4": size === "sm",
-		"size-3": size === "xs",
-	});
+	const copyButtonIconStyle = cn(
+		"h-5 w-5 fill-white",
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/components/molecules/copyButton.tsx]: removed

File: src/components/organisms/settings/profile/deleteModal.tsx [hunk 5]
```diff
@@ -15,12 +15,12 @@ export const DeleteAccountModal = ({ onDelete }: DeleteAccount) => {
 	const { closeModal } = useModalStore();
 
 	return (
-		<Modal name={ModalName.deleteAccount}>
+		<Modal className="" name={ModalName.deleteAccount}>
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/components/organisms/settings/profile/deleteModal.tsx]: removed

File: src/components/organisms/settings/clientConfiguration.tsx [hunk 6]
```diff
@@ -25,72 +25,67 @@ export const ClientConfiguration = () => {
...
+					<Input
+						aria-label={t("hostURL.copyInputAriaLabel")}
+						className="h-12 flex-1 rounded-lg border-gray-950"
+						disabled
+						label={t("hostURL.copyInputAriaLabel")}
+						value={hostURL}
+					/>
+
+					<CopyButton
+						className="ml-2 gap-2 rounded-md hover:bg-gray-1050"
```
Comment (RonenMars) [inline src/components/organisms/settings/clientConfiguration.tsx:76]: Everywhere in the system we use without rounded, so it's a circle. To stay consistent, let's not use `rounded-md` here.
```suggestion
						className="ml-2 gap-2 
 hover:bg-gray-1050"
```

File: src/components/organisms/settings/clientConfiguration.tsx [hunk 7]
```diff
@@ -25,72 +25,67 @@ export const ClientConfiguration = () => {
...
-							value={hostURL}
+							label={t("getToken.copyInputAriaLabel")}
+							value={token}
 						/>
 
 						<CopyButton
-							className="ml-4 rounded-md bg-gray-900 px-3 hover:bg-gray-950"
-							successMessage={t("hostURL.copySuccess")}
-							text={hostURL}
+							className="ml-2 gap-2 rounded-md hover:bg-gray-1050"
```
Comment (RonenMars) [inline src/components/organisms/settings/clientConfiguration.tsx:44]: Everywhere in the system we use without rounded, so it's a circle. To stay consistent, let's not use `rounded-md` here.
```suggestion
							className="ml-2 gap-2  hover:bg-gray-1050"
```

File: src/assets/image/icons/Security.svg [hunk 8]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+<svg width="12" height="16" viewBox="0 0 12 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M11.5 4.10084V9.17899C11.5 9.91993 11.3291 10.6114 11.0072 11.2165C10.6848 11.8225 10.2134 12.3376 9.61287 12.7255L6.17668 14.9458C6.0745 15.0145 5.93945 15.0205 5.82903 14.949L2.38713 12.7255C1.78599 12.3376 1.31506 11.8225 0.992775 11.2165C0.67073 10.6114 0.5 9.92006 0.5 9.17899V4.10084L0.501213 4.08477C0.509586 3.89565 0.662478 3.74912 0.84243 3.7583C0.849347 3.75881 3.27864 3.90891 5.76083 1.10929L5.77746 1.09182C5.90936 0.96289 6.11588 0.970669 6.23868 1.10929C8.58886 3.76021 10.891 3.76659 11.1356 3.75919C11.1479 3.75779 11.1601 3.7569 11.1727 3.7569C11.3535 3.7569 11.5 3.91069 11.5 4.10084Z" stroke-width="0.975177" stroke-linecap="round"/>
```
Comment (RonenMars) [inline src/assets/image/icons/Security.svg:2]: Please add a reference where the SVG has been taken from

File: src/components/organisms/settings/clientConfiguration.tsx (hunk 6 above)
Comment (J1za) [reply src/components/organisms/settings/clientConfiguration.tsx:76]: changed

File: src/components/organisms/settings/clientConfiguration.tsx (hunk 7 above)
Comment (J1za) [reply src/components/organisms/settings/clientConfiguration.tsx:44]: changed

File: src/assets/image/icons/CircleToken.svg [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/assets/image/icons/CircleToken.svg:1]: added

File: src/assets/image/icons/Copy.svg [hunk 10]
```diff
@@ -1,3 +1,3 @@
...
...10829C8.53487 13.5 8.94397 13.3291 9.24557 13.025C9.54718 12.7208 9.7166 12.3083 9.7166 11.8782V10.2985H11.3875C11.8134 10.2996 12.2223 10.1304 12.5245 9.82791C12.8268 9.52541 12.9978 9.11444 13 8.68502V2.12183C12.9988 1.69207 12.8291 1.2802 12.5277 0.976249C12.2263 0.672382 11.8179 0.501106 11.3917 0.5ZM8.42993 11.874C8.42993 11.96 8.39607 12.0425 8.33576 12.1033C8.27538 12.1642 8.19359 12.1983 8.10827 12.1983H1.60831C1.52332 12.1972 1.44214 12.1627 1.38204 12.1021C1.32193 12.0415 1.28773 11.9597 1.28665 11.874V5.31497C1.28665 5.22892 1.32051 5.14644 1.38088 5.08563C1.44119 5.02482 1.52298 4.9906 1.6083 4.9906H8.10826C8.19393 4.98943 8.2764 5.02331 8.33699 5.08433C8.39757 5.14542 8.43109 5.22859 8.42993 5.31498V11.8782L8.42993 11.874ZM11.7133 8.68504C11.7133 8.77109 11.6795 8.85357 11.6191 8.91438C11.5588 8.9752 11.477 9.00941 11.3917 9.00941H9.72073V5.31498C9.72073 4.8848 9.55129 4.47232 9.24969 4.16817C8.94809 3.86403 8.53899 3.69317 8.11248 3.69317H4.57009V2.12183C4.57009 2.03585 4.60395 1.9533 4.66426 1.89249C4.72463 1.83167 4.80643 1.79753 4.89175 1.79753H11.3917C11.4767 1.79862 11.5579 1.83311 11.618 1.89372C11.6781 1.95433 11.7123 2.0362 11.7134 2.12183L11.7133 8.68504Z" />
```
[the remark this answers is not in the document]
Comment (J1za) [reply src/assets/image/icons/Copy.svg:2]: added

File: src/assets/image/icons/Security.svg (hunk 8 above)
Comment (J1za) [reply src/assets/image/icons/Security.svg:2]: added

**Review by RonenMars — APPROVED** [review]

**Pull request merged**